.github/actions/create-cluster/action.yaml

```diff
@@ -10,22 +10,55 @@ inputs:
     required: false
     description: "Kubernetes version that should be used"
     # renovate: datasource=github-releases depName=kubernetes/kubernetes
-    default: "v1.27.3"
+    default: "v1.27.11"
```

Comment:
this is not actually picked up by renovate...

Reply:
not necessarily something to be solved here though

Reply:
will be handled in follow-up